src/applications/my-education-benefits/utils/form-submit-transform.js

```diff
@@ -0,0 +1,393 @@
+// this constant maps the values on address.js in vets.json schema from VA.gov values to LTS values
```

Comment:
In general, these transform methods will generate the expected JSON request that the claim submission DGIB service expects.